app/layout.tsx

```diff
@@ -1,9 +1,38 @@
 import "@/app/globals.css";
+import type { Metadata } from "next";
+import Link from "next/link";
+import { Geist_Mono } from "next/font/google";
+
+export const metadata: Metadata = {
+  title: "swig.dev",
+  description: "minimal event platform for tech and beverage meetups",
+};
```

Comment:
maybe we can skip the metadata for now
kinda think the description here isn't all that great, might not be worth adding it for now
but if you think it's fine, we can change it later